tests/logging_tests/tests.py

```diff
@@ -422,6 +422,30 @@ def test_custom_exception_reporter_is_used(self):
         msg = mail.outbox[0]
         self.assertEqual(msg.body, 'message\n\ncustom traceback text')
 
+    @override_settings(
+        ADMINS=[('whatever admin', 'admin@example.com')],
+    )
+    def test_emit_no_form_tag(self):
+        """
+        #33096 - Emails sent by AdminEmailHandler should not contain form tags
+        in the html version, because that may get it blocked by antivirus.
+        """
+        admin_email_handler = self.get_admin_email_handler(self.logger)
+        old_include_html = admin_email_handler.include_html
+
+        admin_email_handler.include_html = True
+        try:
```

Comment:
We can use a new instance instead of changing `admin_email_handler`:
```suggestion
        handler = AdminEmailHandler(include_html=True)
```